e2e/jest.test-setup.ts

```diff
@@ -12,7 +12,7 @@ const isDebugMode = process.argv.includes('--debug');
  */
 beforeEach(async () => {
   await page.setUserAgent(userAgent);
-  await page.setViewport({width: 1280, height: 680});
+  await page.setViewport({width: 1280, height: 0});
```

Comment:
What is the impact of this change? It's a surprising enough value I would want to see a comment describing why it's appropriate. Even better if this were in a separate PR, unless it's related to the refactors in other files here.